test/core/interface.spec.js

```diff
@@ -22,59 +22,12 @@ describe('interface-ipfs-core tests', () => {
     skip: { reason: 'TODO: DHT is not implemented in js-ipfs yet!' }
   })
 
-  tests.files(defaultCommonFactory, {
-    skip: [
-      {
-        name: 'cp',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'mkdir',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'stat',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'rm',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'read',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'readReadableStream',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'readPullStream',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'write',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'mv',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'flush',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      },
-      {
-        name: 'ls',
-        reason: 'TODO: MFS is not implemented in js-ipfs yet!'
-      }
-    ]
-  })
+  tests.files(defaultCommonFactory)
 
   tests.key(CommonFactory.create({
     spawnOptions: {
       args: ['--pass ipfs-is-awesome-software'],
-      initOptions: { bits: 512 }
+      initOptions: { bits: 1024 }
```

Comment:
This is for faster boot - `defaultCommonFactory` uses 512 also, is this change needed?

Reply:
I ran the tests against the go daemon and it refused to start up with 512 and said it should be 1024 as a minimum.  Can change it back if you prefer.

Reply:
Yep in js-ipfs-api it's set to 1024 it tests against go-ipfs, but these tests only run against js-ipfs and there's no restriction in place currently.

Reply:
There probably should be, for the sake of interop though, right?

Reply:
Yes good point, lets not slow down the tests unnecessarily until that PR comes in though. I'll create an issue for adding it so we can track it.